torchx/cli/cmd_run.py

```diff
@@ -113,6 +113,23 @@ def run(self, args: argparse.Namespace) -> None:
             print(get_builtin_source(builtin_name))
 
 
+class SchedulerArgOnce(argparse.Action):
```

Comment:
Maybe we can reuse this for other commands / options as well?

Reply:
Yes or at least the logic can be used. The same logic can also be added to the _torchxconfig class which is an argparse.Action used for other inputs. This should cover all the cases except the "store_true" cases where the error is less meaningful.